### PR TITLE
Implemented poc_receipts_v2 transaction type and added transactions for blocks module

### PR DESCRIPTION
**Pull request by andreiguzga**
Added transactions_at_height and transactions_at_hash methods for the Blocks module. Also added poc_receipts_v2 transaction type.

### CODE REVIEW

File: src/blocks.rs [hunk 1]
```diff
@@ -19,6 +19,20 @@ pub async fn descriptions(client: &Client, cursor: Option<&str>) -> Result<Descr
...
 }
 
+pub fn transactions_at_height(client: &Client, block: u64) -> Stream<Transaction> {
+    client.fetch_stream(
+        format!("/blocks/{}/transactions", block.to_string()).as_str(),
+        NO_QUERY,
+    )
+}
+
+pub fn transactions_at_hash(client: &Client, hash: &str) -> Stream<Transaction> {
```
Comment (lthiery) [inline src/blocks.rs:29]: `transaction_with_hash` would read better to me

**Review by lthiery — APPROVED** [review]

File: src/blocks.rs (hunk 1 above)
Comment (andreiguzga) [reply src/blocks.rs:29]: Should it be `transaction_with_hash` or `transactions_with_hash`?

Comment (andreiguzga) [reply src/blocks.rs]: That is not the transaction hash. It is the hash of the block. It implements the GET https://api.helium.io/v1/blocks/hash/:hash/transactions from https://docs.helium.com/api/blockchain/blocks. Maybe we should rename hash var to block_hash to make it more evident?

Comment (lthiery) [reply src/blocks.rs]: Ah, woops! My bad.

You're 100% right. This all looks good to me the way it was initially. "at" hash was already right.

Comment (madninja) [reply src/blocks.rs]: rename the _function_ to `transactions_at_block_hash`  to not hide it in the parameter name?

Comment (andreiguzga) [reply src/blocks.rs]: Done.

**Comment by lthiery** [comment]
looks like one clippy warning and we're good to go

**Comment by andreiguzga** [comment]
> 

Is this something I should look into? From the details I can see this

Error: Unable to create clippy annotations! Reason: HttpError: Resource not accessible by integration
Warning: It seems that this Action is executed from the forked repository.
Warning: GitHub Actions are not allowed to create Check annotations, when executed for a forked repos. See https://github.com/actions-rs/clippy-check/issues/2 for details.
Posting clippy checks here instead.

**Comment by madninja** [comment]
@andreiguzga thank you so much for your work here.. well done

**Pull request merged**